### PR TITLE
Fixes #23580: Be more precise in the 8.0.0 release notes

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/23580

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

File: release-data/changelogs/8.0/index.adoc [hunk 1]
```diff
@@ -80,7 +82,7 @@ image::images/yaml.png[]
 
 The YAML format also becomes the internal storage format for techniques (used in the configuration
 git and archives).
-This format is stable, so techniques can be shared between Rudder servers having different versions.
+This format is stable, so techniques can be shared between Rudder servers having higher than 8.0 versions, even if differents.
```
Comment (amousset) [inline release-data/changelogs/8.0/index.adoc:85]:
```suggestion
This format is stable, so techniques can be shared between Rudder servers having higher than 8.0 versions, even if different.
```

File: release-data/changelogs/8.0/index.adoc [hunk 2]
```diff
@@ -19,6 +19,8 @@ evolves.
 
 * Added: Debian 12
 * Now require extended subscription: SLES 11 \<=SP4, SLES 12 \<=SP4, SLES 15 \<=SP2, Windows Server 2008R2, 2012 and 2012R2, Ubuntu 16.04 LTS, Debian 9
+* Rudder agents older than 7.3 will not be supported anymore
```
Comment (amousset) [inline release-data/changelogs/8.0/index.adoc:22]:
```suggestion
* Rudder agents older than 7.3 are not supported on 8.0 servers
```

File: release-data/changelogs/8.0/index.adoc [hunk 3]
```diff
@@ -19,6 +19,8 @@ evolves.
 
 * Added: Debian 12
 * Now require extended subscription: SLES 11 \<=SP4, SLES 12 \<=SP4, SLES 15 \<=SP2, Windows Server 2008R2, 2012 and 2012R2, Ubuntu 16.04 LTS, Debian 9
+* Rudder agents older than 7.3 are not supported on 8.0 servers
+* Rudder agents older 7.0 will not be able to communicate at all with a 8.0.X Rudder server
```
Comment (amousset) [inline release-data/changelogs/8.0/index.adoc:23]:
```suggestion
```

**Pull request merged**